### PR TITLE
[blog] the greatest PL result

### PR DESCRIPTION
**Pull request by bennn**
I think these each need 1-2 sentences definition + justification. Hoping to add that tomorrow, and post on Tuesday?

(I'd wanted to submit this PR much earlier ... but I guess we're still on our 2-weeks-delay schedule for HOPL)

### CODE REVIEW

**Comment by schuster** [comment]
Along with the justification, links to the relevant papers would be great, as well.

File: blog/_src/posts/2017-03-22-1967-greatest-pl-result.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Type Soundness
+_thank you Robin Milner_
+
+
+### Modularization
+_thank you D.L. Parnas_
+
+
+### Generic Programming
+_thank you Jean-Ives Girard and John C. Reynolds_
```
Comment (gasche) [inline blog/_src/posts/2017-03-22-1967-greatest-pl-result.md]: It's Jean-Yves.

**Comment by bennn** [comment]
ok _finally_ updated

@dbp I have a bunch of pdfs in this pull request. 23MB worth. Do you think that's too much?

**Comment by dbp** [comment]
Shouldn't matter -- git can handle that fine, and GitHub pages doesn't have
any limits that I know of. Or did you have another concern in mind?

**Comment by bennn** [comment]
merged https://github.com/nuprl/website/commit/870cedb9e692dcd9e0d3f55eecd3f7d0fe7dd46b

**Comment by mhyee** [comment]
@bennn: @ulysses4ever points out that the author tag is missing.

**Pull request closed without merging**